### ISSUE TITLE
Prediction

### ISSUE THREAD

**Issue by szuhow** [milesial/Pytorch-UNet#519]
After a few adjustments to work on my Mac with M1 and with the dataset in grayscale mode, I managed to achieve the following results after 20 epochs: train loss 0.53152, validation Dice 0.60535, and segmentation results on the images (vein segmentation). I think it would be useful to have an option to set the proportion of BCE to Dice.

![Image](https://github.com/user-attachments/assets/4afd4fd5-0354-435d-b41a-4fd6860d18cf)
![Image](https://github.com/user-attachments/assets/5286af31-27a6-409c-ada7-335a74cf6ffa)
